Formula/qt.rb

```diff
@@ -22,6 +22,7 @@ class Qt < Formula
   depends_on "pkg-config" => :build
   depends_on :xcode => :build
   depends_on :macos => :sierra
+  depends_on "molten-vk"
```

Comment:
Is this a build-time or runtime dependency?

Reply:
A Qt application that is not using Vulkan will run without molten-vk installed. If that is what you are asking.